frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -575,6 +577,7 @@ const defaultHiddenColumns = [
   "primary_mac",
   "public_ip",
   "cpu_type",
+  // TODO: should those be mdm.<blah>?
```

Comment:
With the `accessor` and `id` having either the underscore or the dot, I'm not sure what it should be here.

Reply:
@fleetdm/frontend  ^

Reply:
I found new instances of `mdm_enrollment_status` now that I merged the latest `main` in the PR, I think it'd be best for the frontend team to take it from here for this PR (I think something got merged related to those columns recently or at least after this PR got created), as otherwise it will be a lot of unproductive back and forth /cc @lukeheath .

Reply:
@mna Agreed, we'll take it from here! 

@jacobshandling @RachelElysia Let's work together to make the necessary changes to bring the `mdm.` change together. Almost there!